src/components/ContentPreview/ContentPreview.js

```diff
@@ -378,10 +411,10 @@ class ContentPreview extends PureComponent<Props, State> {
      * @return {void}
      */
     loadPreview = async (): Promise<void> => {
-        const { token: tokenOrTokenFunction, onError, onMetric, collection, ...rest }: Props = this.props;
+        const { token: tokenOrTokenFunction, collection, ...rest }: Props = this.props;
         const { file }: State = this.state;
 
-        if (!this.isPreviewLibraryLoaded() || !file || !tokenOrTokenFunction || this.preview) {
```

Comment:
Removing this guard, can cause the preview to re-show the preview when anything else changes in the component like the sidebar or something. Since loadPreview is called from multiple places, the idea is that it should only re-instantiate preview ---> add listeners ---> call show, if and only if the prior preview instance was destroyed, which only happens when file id changes.

You will have to test.